jerry-debugger/jerry-client-ws.py

```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default=None, type=int,
+                        help="set display range")
+    parser.add_argument("--exception", action="store", default=None, type=int, choices=range(0, 2),
+                        help="set exception config, usage 1: [Enable] or 0: [Disable]:")
```

Comment:
Two minor things:
* No need for range call, just simply: `[0, 1]`
* There is an extra `:` at the end of the help message.